### ISSUE TITLE
The config of celeba-HQ

### ISSUE THREAD

**Issue by pokameng** [lucidrains/denoising-diffusion-pytorch#99]
hello, gays
I used the default parameters directly when I trained with celeba-HQ, but I found that the results were not too good, has anyone had success with celeba-HQ training? Can you share the configuration file please?
Thanks!!!

**Comment by pokameng** [comment]
@kashif @lucidrains

**Comment by Ryhhhh** [comment]
> hello, gays I used the default parameters directly when I trained with celeba-HQ, but I found that the results were not too good, has anyone had success with celeba-HQ training? Can you share the configuration file please? Thanks!!!

@pokameng Have you solved this problem? I also have the same question and would like to ask.

**Comment by winnie-as** [comment]
@pokameng @Ryhhhh Were you able to train with Celeba-HQ dataset? could you also share the loss curve if possible.
